test/mailers/contact_mailer_test.rb

```diff
@@ -0,0 +1,20 @@
+require 'test_helper'
+
+class ContactMailerTest < ActionMailer::TestCase
+  test "contact_us" do
```

Comment:
Although ensuring that the email is pushed to the mail, we should also check that the message is saved to the database, especially since we cannot test mail delivery. I can see about adding that to this test.